jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/JavaxWebSocketServerContainer.java

```diff
@@ -71,34 +80,86 @@ public static WebSocketContainer getWebSocketContainer()
         if (!(handler instanceof ServletContextHandler))
             return null;
 
-        return (javax.websocket.WebSocketContainer)handler.getServletContext().getAttribute("javax.websocket.server.ServerContainer");
+        return (javax.websocket.WebSocketContainer)handler.getServletContext().getAttribute(ServerContainer.class.getName());
+    }
+
+    public static JavaxWebSocketServerContainer ensureContainer(ServletContext servletContext) throws ServletException
+    {
+        ContextHandler contextHandler = ContextHandler.getContextHandler(servletContext);
```

Comment:
This needs a check for when running on Spring / SpringBoot to fail if ContextHandler is null (which it can be there)

Reply:
```
        if (contextHandler == null)	
        {	
            throw new ServletException("Not running on Jetty, Javax Websocket support unavailable");	
        }	

         if (!(contextHandler instanceof ServletContextHandler))	
        {	
            throw new ServletException("Not running in Jetty ServletContextHandler, Javax Websocket support unavailable");	
        }	
```